tests/acceptance/test_shared_issue.py

```diff
@@ -21,23 +18,8 @@ def setUp(self):
         )
         self.login_as(self.user)
 
-    def create_sample_event(self, platform):
-        event = create_sample_event(
-            project=self.project,
-            platform=platform,
-            event_id='d964fdbd649a4cf8bfc35d18082b6b0e',
-            timestamp=1452683305,
-        )
-        event.group.update(
-            first_seen=datetime(2015, 8, 13, 3, 8, 25, tzinfo=timezone.utc),
-            last_seen=datetime(2016, 1, 13, 3, 8, 25, tzinfo=timezone.utc),
-        )
-        return event
-
     def test_cocoa_event(self):
-        event = self.create_sample_event(
-            platform='cocoa',
-        )
+        event = create_sample_event(project=self.project, platform='cocoa')
```

Comment:
You might want to fixate the event_id too so it doesn't get diffed in percy.

Reply:
Oh, is that why we hardcoded that... I don't see any differences in percy so far though?